docs/workload-development-kit/fabric-data-plane.md

```diff
@@ -12,41 +12,98 @@ ms.date: 05/21/2024
+
+The Fabric Workload Development Kit SDK provides a method for acquiring an access token in the workload frontend. An example of usage can be found in the frontend [Sample Workload Controller](https://github.com/microsoft/Microsoft-Fabric-workload-development-sample/blob/main/Frontend/src/controller/SampleWorkloadController.ts).
+
+ This client token must be passed to the workload backend and exchanged using the on-behalf-of flow for a token with the necessary scopes to access your desired resource, such as OneLake. For example, in order to access and read from a Lakehouse, a user must authorize the application to make API calls on their behalf using the Azure Storage user_impersonation permission. Then, in the backend, the access token must be obtained with the delegated scope `https://storage.azure.com/user_impersonation` in order to use Azure Storage.
+
+If you decide to use SQL to access your customer data, the access token must be obtained with the scope `https://database.windows.net//user_impersonation` in order to use Azure SQL Database and the Microsoft.Data.SqlClient namespace. The access token can be obtained without the extra forward slash in the scope from Azure SQL Database, *however* the access token won't be validated by the SQLConnection class, and so we recommend using the scope as written here.
+For more examples of token authentication, refer to the Microsoft Fabric Developer kit sample.
 
-The Fabric Workload Development Kit SDK provides a method for acquiring an access token in the workload frontend. This client token must be passed to the workload backend and exchanged using the on-behalf-of flow for a token with the necessary scopes to access your desired resource, such as OneLake. For example, in order to access and read from a Lakehouse, a user must have authorized the application to make API calls on their behalf using the Azure Storage user_impersonation permission. Then, the access token must be obtained with the delegated scope `https://storage.azure.com/user_impersonation` in order to use Azure Storage. 
+More details on how to obtain a token can be found in the [Microsoft Fabric Workload Development REST API documentation](https://go.microsoft.com/fwlink/?linkid=2271986).
 
-For examples of token authentication, please refer to the Microsoft Fabric Developer kit sample.
+### Read Metadata
 
-More details on how to obtain a token can be found in the [Microsoft Fabric Workload Development REST API documentation](https://go.microsoft.com/fwlink/?linkid=2271986). 
+Fabric REST APIs also provide you with a way to access the different properties of items. For example, querying the [Get Lakehouse API](/rest/api/fabric/lakehouse/items/get-lakehouse?tabs=HTTP) provides you with the metadata for a certain Lakehouse, including useful properties such as OneLake paths and the SQL connection string.
```

Comment:
'Fabric REST APIs also provide you with a way to access the different properties of items.' --> 'Fabric REST APIs provide a way to access item properties.'

'/rest/api/fabric/lakehouse/items/get-lakehouse?tabs=HTTP' --> '/rest/api/fabric/lakehouse/items/get-lakehouse'.